src/main/resources/templates/teams.html

```diff
@@ -95,10 +120,13 @@ <h2 class="team-heading team-heading-current-team" th:text="#{team.heading.reque
                                    <td th:text="${team.value.description}">Description</td>
                                    <td><a href="#" class="btn btn-standard" th:href="@{/withdraw/{teamId}(teamId=${team.value.id})}"><i class="fa fa-minus-circle btn-fa-red"></i> Withdraw</a></td>
                                 </tr>
+                                <!--<tr>-->
+                                    <!--<td height="700"></td>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)